packages/telegraf/extra/dcos-telegraf.service

```diff
@@ -16,5 +16,6 @@ EnvironmentFile=/opt/mesosphere/etc/telegraf.env
 
 LimitNOFILE=16384
 
+ExecStartPre=/bin/ping -c1 leader.mesos
```

Comment:
I dont have the full context but this seems like a roundabout way of saying mesos agent requires telegraf to be up and running first.

Why not add a `ExecStartPre` to mesos agent service that checks for the presence of `/run/dcos/telegraf/dcos_statsd.sock ` instead ? cc @jongiddy

Reply:
I'll create a PR for that. We already discussed it with @jongiddy https://mesosphere.slack.com/archives/GQD0R41RD/p1596540642317000